lib/plans/checkout.dart

```diff
@@ -156,28 +157,36 @@ class _CheckoutState extends State<Checkout>
     }
   }
 
+  Future<void> openDesktopWebview() async {
+    String os = Platform.operatingSystem;
+    Locale locale = Localizations.localeOf(context);
+    final format = NumberFormat.simpleCurrency(locale: locale.toString());
+    final currencyName = format.currencyName ?? "USD";
+    final redirectUrl = await sessionModel.paymentRedirect(
+      widget.plan.id,
+      currencyName,
+      emailController.text,
+      "stripe",
+      os,
+    );
+    if (!Platform.isMacOS && !Platform.isWindows) {
```

Comment:
I think we can simplify this as 

```
 if (Platform.isLinux) {
      await context.pushRoute(AppWebview(url: redirectUrl));
    } else if (Platform.isWindows) {
      await AppBrowser.openWindowsWebview(redirectUrl);
    } else {
      // Macos
      final browser = AppBrowser(onClose: checkProUser);
      await browser.openMacWebview(redirectUrl);
    }
```

Reply:
Thanks! I actually wanted to use a switch statement here. Just pushed that change

Reply:
I am fine with that as well